incubator/neo4j/README.md

```diff
@@ -0,0 +1,72 @@
+# Neo4j
+
+[Neo4j](https://neo4j.com/) is a highly scalable native graph database that leverages data relationships as first-class entities, helping enterprises build intelligent applications to meet today’s evolving data challenges.
+
+## TL;DR;
+
+```bash
+$ helm install incubator/neo4j
+```
+
+## Introduction
+
+This chart bootstraps a [Neo4j](https://github.com/neo4j/docker-neo4j) deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+
+## Prerequisites
+
+- Kubernetes 1.6+ with Beta APIs enabled
+- PV provisioner support in the underlying infrastructure
+
+## Installing the Chart
+
+To install the chart with the release name `neo4j-helm`:
+
+```bash
+$ helm install --name neo4j-helm incubator/neo4j
+```
+
+The command deploys Neo4j on the Kubernetes cluster in the default configuration. The [configuration](#configuration) section lists the parameters that can be configured during installation.
+
+> **Tip**: List all releases using `helm list`
+
+## Uninstalling the Chart
+
+To uninstall/delete the `neo4j-helm` deployment:
+
+```bash
+$ helm delete neo4j-helm --purge
+```
+
+The command removes all the Kubernetes components associated with the chart and deletes the release.
+
+## Configuration
+
+The following tables lists the configurable parameters of the Neo4j chart and their default values.
+
```

Comment:
I've updated it with the things I think users might want to use. Would you usually list every single variable in the table?